README.md

```diff
@@ -64,6 +67,27 @@ If you want to see the logs generated by test, run the test as:
 $ SHOW_LOG=--show-job-log make test
 ```
 
+<a name="releasing-the-b2d-iso"></a>
+## Releasing the minishift-b2d ISO
+
+<a name="release-prerequisites"></a>
+### Prerequisites
+
+- Install [Go](https://en.wikipedia.org/wiki/Go_(programming_language)) with `GOPATH` properly set.
```

Comment:
"Install" is a verb, you need a noun to fit this enumeration. 

Working installation of ....